### DISCUSSION TITLE
Upgraded Power and Heart Rate Charts

### DISCUSSION THREAD

**Discussion by wbjenn** [cagnulein/qdomyos-zwift#371] [Ideas]
Roberto,

How difficult is it to create Power and Heart Rate Charts that reflect the users zones and show the zones distinct from one another rather than the gradation that is now used? Of these two charts, the Power Chart is the most important for me. Most of your users currently use the app with bikes that don't have auto-resistance. A true Power Chart (even better if it has the blue line to reflect target zones) would probably be beneficial to more users.

Thanks,
Brad

**Comment by cagnulein** [comment]
i know, but i hate doing UI development :D but yes it's doable but requires time. Maybe when i will be in vacation this summer :)

**Reply by wbjenn** [reply]
Typical dev! ;) If it has to wait, I do like the idea of the Puglian Power Chart!
